### DISCUSSION TITLE
When will create the branch langdale

### DISCUSSION THREAD

**Discussion by junzhuimx** [Freescale/meta-freescale#1329] [Q&A] [answered]
As Yocto 4.1 Langdale is announced in this Oct, I recognize that the branch "langdale" does not create in our layer

**Comment by otavio** [comment, marked as answer]
@junzhuimx yes, but only when disruptive changes start to happen ... there are some I am planning to do, but I still need to start working on those.

For now, I'd prefer to postpone splitting the branches as no incompatible modifications have been done yet.
